server/s3_get_bucket_action_v2.cc

```diff
@@ -63,6 +63,24 @@ void S3GetBucketActionV2::validate_request() {
 
   std::shared_ptr<S3ObjectListResponseV2> obj_v2_list =
       std::dynamic_pointer_cast<S3ObjectListResponseV2>(object_list);
+  std::string expected_owner;
```

Comment:
why not initialize expected_owner here only ? we can combine line 66 and 67 !

Reply:
ok